src/org/zaproxy/zap/extension/zest/ZestZapUtils.java

```diff
@@ -431,6 +432,67 @@ public static String toUiString(ZestElement za, boolean incParams,
 						.getString("zest.element.assign.calc.title");
 			}
 
+		} else if (za instanceof ZestAssignFromElement) {
+			ZestAssignFromElement zsa = (ZestAssignFromElement) za;
+			if (incParams) {
+
+				StringBuilder sb = new StringBuilder();
+				if(!zsa.isFilteredByElementName() && !zsa.isFilteredByAttribute()){
+					sb.append("Elements()");
+				}
+
+				if(zsa.isFilteredByElementName() || zsa.isFilteredByAttribute()) {
+					sb.append("where(");
+				}
+
+				if(zsa.isFilteredByElementName()){
+					sb.append("Element == '");
+					sb.append(zsa.getElementNameFilter());
+					sb.append("'");
+				}
+
+				if(zsa.isFilteredByElementName() && zsa.isFilteredByAttribute()){
+					sb.append(" && ");
+				}
+
+				if(zsa.isFilteredByAttribute()){
+					sb.append("Attributes['");
+					sb.append(zsa.getAttributeNameFilter());
+					sb.append("'] match '");
+					sb.append(zsa.getAttributeValueFilter());
+					sb.append("'");
+				}
+
+				if(zsa.isFilteredByElementName() || zsa.isFilteredByAttribute()) {
+					sb.append(")");
+				}
+
+				if(zsa.areFilteredElementsReversed()){
+					sb.append(".Reverse()");
+				}
+
+				sb.append("[");
+				sb.append(zsa.getElementIndex());
+				sb.append("]");
+
+				if(zsa.isReturningElement()){
+					sb.append(".SelectContent()");
+				}
+
+				if(zsa.isReturningAttribute()){
+					sb.append(".SelectAttribute('");
+					sb.append(zsa.getReturnedAttributeName());
+					sb.append("')");
+				}
+
+				return indexStr + MessageFormat.format(
+						Constant.messages.getString("zest.element.assign.fromElement"),
```

Comment:
Use `I18N.getString(String, Object...)` instead of `MessageFormat`.
Worth noting that the above strings should also be internationalised.